Project/Source/Resources/ResourceVideo.cpp

```diff
@@ -0,0 +1,10 @@
+#include "ResourceVideo.h"
+#include "Utils/Logging.h"
+
+void ResourceVideo::Load() {
+	std::string filePath = GetResourceFilePath();
+	LOG("Loading video from path: \"%s\".", filePath.c_str());
```

Comment:
Is the video being streamed from disk while playing? Just asking.

Reply:
I think it doesnt.
Load() loads the file into libAV and fills various datastructures, then, from there we can read streams (video, audio, etc) , and from there we can read each frame, so i think it is stored in RAM... Maaaaaaybe? Not sure

Reply:
OK, make sure that if anything is getting stored in RAM, it gets loaded in this method.

Reply:
Okey!
I'll take a look for that when i make the next PR of video